packages/user-setup/user-setup.0.7/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+homepage: "https://github.com/AltGr/opam-user-setup"
+bug-reports: "https://github.com/AltGr/opam-user-setup/issues"
+license: "ISC"
+tags: ["org:ocamlpro" "flags:plugin"]
+dev-repo: "https://github.com/AltGr/opam-user-setup.git"
```

Comment:
Sorry for asking, but are these urls not pointing to the wrong repository?
I would have expected the urls to point to OCamlPro/opam-user-setup, i.e. the same as the location of the archive file.

Reply:
Indeed, fixed in the incoming PR